### PR TITLE
Format options for custom text fields. [SDESK-4805]

### PR DESCRIPTION
**Pull request by GyanP**

### CODE REVIEW

File: scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts [hunk 1]
```diff
@@ -126,10 +126,9 @@ export function ContentProfileSchemaEditor(vocabularies) {
 
             scope.getEditor3FormattingOptions = (fieldName) => {
                 const isCustomPlainTextField = typeof scope.fields[fieldName] === 'object'
-                    && typeof scope.fields[fieldName].field_options === 'object'
-                    && scope.fields[fieldName].field_options.single === true;
```
Comment (tomaskikutis) [inline scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts]: What does `field_options.single` mean?

File: scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts [hunk 2]
```diff
@@ -126,10 +126,9 @@ export function ContentProfileSchemaEditor(vocabularies) {
...
+                    && scope.fields[fieldName].field_type === 'text';
```
Comment (tomaskikutis) [inline scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts]: What other `field_type`s there are?

File: scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts (hunk 1 above)
Comment (GyanP) [reply scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts]: Setting `field_options.single` to `true` means that the field is a single-line field. eg: slugline is a single-line field.

File: scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts (hunk 2 above)
Comment (GyanP) [reply scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts]: `related_content`, `media`, `date_field` etc.

Comment (tomaskikutis) [reply scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts]: > related_content, media, date_field etc.

None of these can use editor3, does it make sense to check them?

Comment (GyanP) [reply scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts]: Not at all, we have to check it for `text` fields only.

**Pull request merged**